VideoAPI/Testing.Common/Assertions/AssertConferenceForAdminResponse.cs

```diff
@@ -18,7 +18,7 @@ public static void ForConference(ConferenceForAdminResponse conference)
             conference.Id.Should().NotBeEmpty();
             conference.HearingRefId.Should().NotBeEmpty();
             conference.Status.Should().NotBeNull();
-            //conference.TelephoneConferenceId.Should().NotBeEmpty();
+            conference.CreatedDateTime.Should().NotBe(DateTime.MinValue);
```

Comment:
Don't think we need this one Valentina. For existing hearings the value should be null. Otherwise the created date.

Reply:
Yes, agreed, done